lib/ffmpeg/encoderbase.py

```diff
@@ -89,7 +90,8 @@ def gen_output_opts(self):
 
     # WA: LDB is not enabled by default for HEVCe on gen11+, yet.
     if get_media()._get_gpu_gen() >= 11 and self.codec.startswith("hevc"):
-      opts += " -b_strategy 1"
+      if vars(self).get("default", None) is None:
+        opts += " -b_strategy 1"
```

Comment:
Hmmm... introducing a workaround for another workaround.  Also, a variable called `default` is a bit too broad in meaning.  Clearly we should be able to find a better approach than this.